Running jmh tests fails with `error: plug-in not found: ErrorProne`

JMH tests fails for latest master (d705aa8ccb3aaca4c4eb0fef74658fed1cac3e83)
```
$ ./gradlew :iceberg-spark:jmh -PjmhIncludeRegex=SparkParquetReadersFlatDataBenchmark -PjmhOutputPath=benchmark/spark-parquet-readers-flat-data-benchmark-result.txt

.
.
.

> Task :iceberg-spark:jmhCompileGeneratedClasses FAILED
error: plug-in not found: ErrorProne

FAILURE: Build failed with an exception.
```

I found a relevant comment regarding interaction of errorprone and jmh at https://github.com/tbroyer/gradle-errorprone-plugin/issues/3#issuecomment-388853116

Should we disable error-prone for jmh generated classes? This can be done using
```
jmhCompileGeneratedClasses {
  options.errorprone.enabled = false
}
```

--- a/jmh.gradle
+++ b/jmh.gradle
@@ -30,5 +30,9 @@ configure(jmhProjects) {
     humanOutputFile = file(jmhOutputPath)
     include = [jmhIncludeRegex]
   }
+
+  jmhCompileGeneratedClasses {
+    options.errorprone.enabled = false
+  }
 }
 

